src/ansys/fluent/core/__init__.py

```diff
@@ -106,10 +106,10 @@ def version_info() -> str:
 pydoc.text.docother = fldoc.docother.__get__(pydoc.text, pydoc.TextDoc)
 
 # Whether to use datamodel state caching
-DATAMODEL_USE_STATE_CACHE = True
+DATAMODEL_USE_STATE_CACHE = False
```

Comment:
@prmukherj Is there an issue that necessitates this change? If so, then we need to investigate why.

Reply:
@seanpearsonuk, no. Just making sure that caching is not playing a role here as I was facing some issues while local testing. Will revert it soon and merge.